core/ui/TagTemplate.tid

```diff
@@ -27,4 +33,4 @@ color:$(foregroundColor)$;
 </span>
 \end
 
-<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>
```

Comment:
@Infurnoape, Some templates do not have a newline at the end of the last line for a reason. This may or may not be one of them. The effect, when it happens, is extra space within the element.